### PR TITLE
how to synthesise EAR JWTs

### PR DESCRIPTION
**Pull request by thomas-fossati**

### CODE REVIEW

File: play-by-ear.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+See [installation instructions](https://smallstep.com/docs/step-cli/installation) for the most common platforms.
+
+(TL;DR -- On MacOSX, do `brew install step`.)
+
+### Obtain the `veraison/ear` examples
+
+* Clone Veraison's EAR repo:
+```sh
+git clone github.com/veraison/ear
```
Comment (paulhowardarm) [inline play-by-ear.md:17]: I don't think this works exactly as written without the explicit `https://` on the front of the GitHub URL, at least it didn't for me, but possibly it depends on what shell you're using.

**Review by paulhowardarm — CHANGES_REQUESTED** [review]
Worked like a charm, thank you! Requesting one tiny change, just because one of the shell commands didn't work for me when I pasted it verbatim.

File: play-by-ear.md (hunk 1 above)
Comment (thomas-fossati) [inline play-by-ear.md:17]:
```suggestion
git clone https://github.com/veraison/ear
```

Comment (thomas-fossati) [reply play-by-ear.md]: Thanks for spotting this!

Comment (thomas-fossati) [reply play-by-ear.md]: Fixed in [a2ad0fd](https://github.com/CCC-Attestation/attested-tls-poc/pull/22/commits/a2ad0fda4ae59862a385aef644da74649a52ce73)

**Review by paulhowardarm — APPROVED** [review]
LGTM - thanks for the tweaks

**Pull request merged**